server/datastore/s3/installer.go

```diff
@@ -31,24 +31,14 @@ func NewInstallerStore(config config.S3Config) (*InstallerStore, error) {
 	return &InstallerStore{s3store}, nil
 }
 
-// Exists checks if an installer exists in the S3 bucket
-func (i *InstallerStore) Exists(ctx context.Context, installer fleet.Installer) (bool, error) {
-	key := i.keyForInstaller(installer)
-	_, err := i.s3client.HeadObject(&s3.HeadObjectInput{Bucket: &i.bucket, Key: &key})
-	if err != nil {
-		return false, err
-	}
-	return true, nil
-}
-
 // Get retrieves the requested installer from S3
-func (i *InstallerStore) Get(ctx context.Context, installer fleet.Installer) (io.ReadCloser, error) {
+func (i *InstallerStore) Get(ctx context.Context, installer fleet.Installer) (*io.ReadCloser, *int64, error) {
```

Comment:
Why a pointer to an interface?, I'd just return `(io.ReadCloser, int64, error)`.

Reply:
ya, makes sense! fixed